packages/medusa/src/api/routes/admin/order-edits/index.ts

```diff
@@ -104,13 +104,72 @@ export default (app) => {
   return app
 }
 
+/**
+ * @schema AdminOrderEditsRes
+ * type: object
+ * properties:
+ *   order_edit:
+ *     $ref: "#/components/schemas/OrderEdit"
+ */
 export type AdminOrderEditsRes = {
   order_edit: OrderEdit
 }
+
+/**
+ * @schema AdminOrderEditsListRes
+ * type: object
+ * properties:
+ *   order_edits:
+ *     type: array
+ *     $ref: "#/components/schemas/OrderEdit"
```

Comment:
The original OAS for this one seems to be incorrect. This should be under the `items` property since `order_edits` is an array

Reply:
Good catch! Fixed in 5d881002aa61b2400955f2d0adcbbec8ebede3d8